src/components/proposals/version-history.vue

```diff
@@ -43,29 +43,29 @@ export default {
-      q-icon.absolute(v-if="versionHistory.masterpolicy[0].system_originalApprovedDate_t" name="fas fa-arrow-right" :style="{ 'right': '20px', 'top': '28px' }")
-      div(:style="{ 'border-radius': '50%', 'background': '#242F5D', 'width': '30px', 'height': '30px', 'display': 'flex', 'justify-content': 'center', 'align-items': 'center' }" :class="{ 'selected-icon': !versionHistory.masterpolicy[0].system_originalApprovedDate_t }")
-        q-icon(name="fas fa-check" :color="versionHistory.masterpolicy[0].system_originalApprovedDate_t ? 'white' : versionHistory.state === PROPOSAL_STATE.REJECTED ? 'negative' : 'primary'")
-      .h-h5.text-weight-700.q-mt-sm.q-mb-md(:class="{ 'selected-title': !versionHistory.masterpolicy[0].system_originalApprovedDate_t }") {{ $t('proposals.version-history.original') }}
-      div {{ parsedDate(versionHistory.masterpolicy[0].system_originalApprovedDate_t) }}
-    .bg-internal-bg.q-mr-md.q-mb-md.rounded-border.q-pa-md(v-else :style="{ 'min-width': '160px', 'position': 'relative' }" :class="{ 'selected-card': !versionHistory.masterpolicy.length, 'rejected-card': versionHistory.state === PROPOSAL_STATE.REJECTED }")
-      div(:style="{ 'border-radius': '50%', 'background': '#242F5D', 'width': '30px', 'height': '30px', 'display': 'flex', 'justify-content': 'center', 'align-items': 'center' }" :class="{ 'selected-icon': !versionHistory.masterpolicy.length }")
-        q-icon(name="fas fa-check" :color="versionHistory.masterpolicy.length ? 'white' : versionHistory.state === PROPOSAL_STATE.REJECTED ? 'negative' : 'primary'")
-      .h-h5.text-weight-700.q-mt-sm.q-mb-md(:class="{ 'selected-title': !versionHistory.masterpolicy.length }") {{ $t('proposals.version-history.original') }}
-      div(v-if="versionHistory.state === PROPOSAL_STATE.PROPOSED") {{ $t('proposals.version-history.currentOnVoting') }}
-      div(v-else-if="versionHistory.state === PROPOSAL_STATE.REJECTED") {{ $t('proposals.version-history.rejected') }}
-      div(v-else) {{ parsedDate(versionHistory.originalApprovedDate) }}
-    .bg-internal-bg.q-mr-md.q-mb-md.rounded-border.q-pa-md(v-if="versionHistory.masterpolicy.length" :style="{ 'min-width': '160px', 'position': 'relative' }" :class="{ 'selected-card': versionHistory.masterpolicy[0].system_originalApprovedDate_t, 'rejected-card': versionHistory.state === PROPOSAL_STATE.REJECTED }")
-      div(:style="{ 'border-radius': '50%', 'background': '#242F5D', 'width': '30px', 'height': '30px', 'display': 'flex', 'justify-content': 'center', 'align-items': 'center' }" :class="{ 'selected-icon': versionHistory.masterpolicy[0].system_originalApprovedDate_t }")
+    .bg-internal-bg.q-mr-md.q-mb-md.rounded-border.q-pa-md(v-if="versionHistory?.masterpolicy.length" :style="{ 'min-width': '160px', 'position': 'relative' }" :class="{ 'selected-card': !versionHistory?.masterpolicy[0].system_originalApprovedDate_t, 'rejected-card': versionHistory?.masterpolicy[0].details_state_s === PROPOSAL_STATE.REJECTED }")
```

Comment:
@arsenijesavic  this is really messy

Reply:
@Alex-IS Why? After all, the template loads faster than apollo's response, which causes errors in the console, I think adding an extra check is a good way to deal with errors